tox.ini

```diff
@@ -18,6 +19,7 @@ python =
     3.7: py37
     3.8: py38
     3.9: py39
+    3.10: py310
```

Comment:
I tried this on my fork, and it didn't work... did you manage to get this working locally?

https://github.com/adehad/jira/actions/runs/2044892461

Reply:
I'll back this out for now. Maybe whether a separate PR.